throughput/spam.go

```diff
@@ -71,29 +71,30 @@ type Spammer struct {
 	numAccounts int
 }
 
-func NewSpammer(
-	uris []string,
-	key *auth.PrivateKey,
-	balance uint64,
-	sZipf, vZipf float64,
-	txsPerSecond, minTxsPerSecond, txsPerSecondStep, numClients, numAccounts int,
-) *Spammer {
+func NewSpammer(sc *SpamConfig,
+	sh SpamHelper,
+) (*Spammer, error) {
```

Comment:
nit: could we make this consistent by either putting both params on a single line (there's not that many anymore) or give each their own line?

Reply:
oops, yea this should be on its own line